Paco-Server/src/com/google/sampling/experiential/server/CloudSQLDaoImpl.java

```diff
@@ -196,6 +202,12 @@ public boolean insertEvent(Event event) throws SQLException, ParseException {
     }
     return retVal;
   }
+  
+  @Override
+  public List<EventDAO> getEvents(String eventId) throws SQLException, ParseException{
```

Comment:
it is possible to create a SQL injection attack here when constructing a string with eventId as a string.